site/book/01-getting-started/02-quickstart.md

```diff
@@ -86,14 +99,39 @@ You can preview the changes that will be made to the cluster:
 $ kpt live preview
 ```
 
-Finally, apply the resources to the cluster:
+Apply the resources to the cluster:
 
 ```shell
 $ kpt live apply --reconcile-timeout=15m
 ```
 
 This waits for the resources to be reconciled on the cluster by monitoring their status.
 
+## Fetch updates
+
+On Day n, there might be updates to the remote `nginx` package which you want to rebase your local package against.
+A typical `git rebase` might lead to merge conflicts in this scenario. `kpt pkg update` is semantic-aware
```

Comment:
semantic-aware -> schema-ware (I'm changing this everywhere).

Reply:
Updated.